extensions/3rdparty/LyricWiki/Special_Soapfailures.php

```diff
@@ -171,34 +177,33 @@ function execute( $par ){
 			$wgOut->addWikiText($msg);
 
 			// Form for clearing a fixed song.
-			$wgOut->addHTML(wfMsg('soapfailures-mark-as-fixed') . "
+			$wgOut->addHTML(wfMsg($this->I18N_PREFIX.'-mark-as-fixed') . "
 							<form method='post'>
-								".wfMsg('soapfailures-artist')." <input type='text' name='artist'/><br/>
-								".wfMsg('soapfailures-song')." <input type='text' name='song'/><br/>
-								<input type='submit' name='fixed' value='".wfMsg('soapfailures-fixed')."'/>
+								".wfMsg($this->I18N_PREFIX.'-artist')." <input type='text' name='artist'/><br/>
+								".wfMsg($this->I18N_PREFIX.'-song')." <input type='text' name='song'/><br/>
+								<input type='submit' name='fixed' value='".wfMsg($this->I18N_PREFIX.'-fixed')."'/>
 							</form><br/>");
 
-			$data = $wgMemc->get($CACHE_KEY_DATA);
-			$cachedOn = $wgMemc->get($CACHE_KEY_TIME);
-			$statsHtml = $wgMemc->get($CACHE_KEY_STATS);
+			$data = $wgMemc->get($this->CACHE_KEY_DATA);
+			$cachedOn = $wgMemc->get($this->CACHE_KEY_TIME);
+			$statsHtml = $wgMemc->get($this->CACHE_KEY_STATS);
 			if(!$data){
-				$db = &wfGetDB(DB_SLAVE)->getProperty('mConn');
-				$queryString = "SELECT * FROM lw_soap_failures ORDER BY numRequests DESC LIMIT $MAX_RESULTS";
-				if($result = mysql_query($queryString,$db)){
-					$data = array();
-					if(($numRows = mysql_num_rows($result)) && ($numRows > 0)){
-						for($cnt=0; $cnt<$numRows; $cnt++){
-							$row = array();
-							$row['artist'] = mysql_result($result, $cnt, "request_artist");
-							$row['song'] = mysql_result($result, $cnt, "request_song");
-							$row['numRequests'] = mysql_result($result, $cnt, "numRequests");
-							$row['lookedFor'] = mysql_result($result, $cnt, "lookedFor");
-							$row['lookedFor'] = formatLookedFor($row['lookedFor']);
-							$data[] = $row;
-						}
+				$queryString = "SELECT * FROM {$this->TABLE_NAME} ORDER BY numRequests DESC LIMIT $MAX_RESULTS";
```

Comment:
Specified columns, changed to ->select() and tested on devbox... will be in the 2nd additional commit.